go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -358,7 +358,15 @@ func (qre *QueryExecutor) checkAccess(authorized *tableacl.ACLResult, tableName
 }
 
 func (qre *QueryExecutor) execDDL() (*sqltypes.Result, error) {
-	ddlPlan := planbuilder.DDLParse(qre.query)
+	sql := qre.query
+	var err error
+	if qre.plan.FullQuery != nil {
```

Comment:
Why is this change necessary -- it seems like you should be able to pass through the original query unmodified rather than sending the generated full query, no?